composer.json

```diff
@@ -205,7 +211,7 @@
         "drush/drush": "^10",
         "enyo/dropzone": "4.3.0",
         "guzzlehttp/guzzle": "^6.3",
-        "massgov/mayflower-artifacts": "9.49.1",
+        "massgov/mayflower-artifacts": "dev-patternlab/DP-17483_dataviz-textwrapping-container-suggestions",
```

Comment:
Adding a comment here to keep track on this change

Reply:
The MF version is changed back to 9.49.1.